library/Pdfexport/HeadlessChrome.php

```diff
@@ -240,111 +243,168 @@ public function fromHtml($html, $asFile = false)
     }
 
     /**
-     * Export to PDF
+     * Tries to asynchronously generate/print a PDF raw string from the configured browser.
```

Comment:
That's a description. The first line in doc blocks should be a statement.

Also:

* it doesn't print (you document a method of a class, in which case print means echo)
* Since it doesn't print nor writes to disk, the fact that it results in a PDF *string* is self explaining
* I don't see a reason to document *here* that a browser is used. If at all, that's something for the class docblock